readme/simplesamlphp-setup.md

```diff
@@ -4,12 +4,21 @@ To configure SimpleSAMLphp, perform the following steps after initially setting
 
 1. Execute `blt simplesamlphp:init`. This will perform the initial setup tasks including:
 	* Adds the simplesamlphp_auth module as a project dependency.
-  * Patches your .htaccess file to allow access to the simplesaml path.
 	* Copies configuration files to `${project.root}/simplesamlphp`
 	* Adds a simplesamlphp property to project.yml
 	* Creates a symbolic link in the docroot to the web accessible directory of the simplesamlphp library.
 	* Adds a settings.php file to the project's default settings directory.
-
+	
+1. Edit `docroot/.htaccess` to include the following 2 lines. Note: the only 2 lines that need to be added are the lines marked with "+" signs. 
```

Comment:
Seeing this now. We modified our VCL to pass paths beginning with /simplesaml since we do IDP SSO. Has been working for us.